app/src/main/res/values/strings.xml

```diff
@@ -306,60 +306,60 @@
+    <string name="settings_https_heading">Set up HTTP(S) proxy</string>
+    <string name="settings_https_desc">This setting is only a recommendation and may be ignored by apps, as those using HTTP(S) proxy bypass both firewall and DNS rules.</string>
 
     <string name="orbot">Orbot</string>
-    <string name="orbot_socks5">TCP-only Proxy</string>
+    <string name="orbot_socks5">TCP-only proxy</string>
     <string name="orbot_socks5_desc">Forwards all TCP connections to Orbot.</string>
     <string name="orbot_http">HTTP Proxy</string>
     <string name="orbot_http_desc">Recommends that apps use Orbot\'s HTTP proxy.</string>
-    <string name="orbot_both">TCP + HTTP Proxy</string>
+    <string name="orbot_both">TCP + HTTP proxy</string>
     <string name="orbot_both_desc">All of the above.</string>
     <string name="orbot_none">None (default)</string>
     <string name="orbot_none_desc">None of the above.</string>
 
     <string name="orbot_bs_status_1">Forwarding %1$s connections to Orbot.</string>
     <string name="orbot_bs_status_2">HTTP Proxy is set (apps may ignore it).</string>
     <string name="orbot_bs_status_3">Forwarding %1$s to Orbot, HTTP Proxy also set.</string>
-    <string name="orbot_bs_status_4">One-click Tor-as-a-Proxy (Orbot) setup.</string>
+    <string name="orbot_bs_status_4">One-click Tor-as-a-proxy (Orbot) setup.</string>
 
     <string name="orbot_status_arg_2">TCP</string>
     <string name="orbot_status_arg_3">DNS and TCP</string>
 
     <string name="orbot_app_issue">Could not launch Orbot. Is it installed?</string>
     <string name="orbot_bs_status_trying_connect">Setting up Orbot…</string>
 
-    <string name="orbot_stop_dialog_title">Manual intervention required</string>
+    <string name="orbot_stop_dialog_title">Action needed</string>
     <string name="orbot_stop_dialog_message_combo">%1$s\n\n%2$s</string>
-    <string name="orbot_stop_dialog_message">RethinkDNS has stopped forwarding connections to Orbot, but Orbot maybe running in the background.</string>
-    <string name="orbot_stop_dialog_dns_message">RethinkDNS is setup to forward DNS queries to Orbot. Change as required.</string>
+    <string name="orbot_stop_dialog_message">RethinkDNS has stopped forwarding connections to Orbot, but Orbot may be running in the background.</string>
+    <string name="orbot_stop_dialog_dns_message">RethinkDNS is set up to forward DNS queries to Orbot. Change as required.</string>
     <string name="orbot_stop_dialog_negative">Stop Orbot</string>
     <string name="orbot_stop_dialog_neutral">Change DNS</string>
     <string name="orbot_stop_dialog_positive">Dismiss</string>
 
     <string name="orbot_title">Orbot and Proxies</string>
     <string name="orbot_explanation">
             Orbot is a free proxy app that empowers other apps to use the Internet more &lt;u>securely&lt;/u>.
```

Comment:
```suggestion
            Orbot is a free proxy app that lets other apps to use the Internet more &lt;u>securely&lt;/u>.
```
"lets" used in other similar strings.

Reply:
this text is used verbatim from Orbot developers.